source/help/messaging/mentioning-teammates.rst

```diff
@@ -50,6 +50,24 @@ You can ignore channel-wide mentions in specific channels in the **Channel Menu
 .. code-block:: none
 
   @here can someone do a quick review of this?
+  
+@groupname (Beta)(E20)
+~~~~~
+
+System Admins can enable mentions for `LDAP synced groups <https://docs.mattermost.com/deployment/ldap-group-sync.html>`_ via the Group Configuration page.
+
+**Note**: This functionality is not yet supported on mobile.
+
+Once enabled for a specific group, users will be able to mention and notify the entire group (similar to @channel or @all). Every member of the group who is also present in the channel the mention was posted, will receive a notification. If members of the group mentioned are not members of the channel, the user who posted the mention is prompted to invite them.
+
+Group mention identifiers (slugs) use the LDAP group name by default, but they can be customized and renamed from the group configuration page.
```

Comment:
```suggestion
Group mention identifiers (slugs) use the LDAP group name by default. You can customize/rename the slug via **System Console > Groups (Beta)**. Select **Edit** next to the group you want to edit.
```
Question here: Following the instructions I've added, I can't see where to change the slug. When clicking **Edit** the group profile section where the name is, is greyed out. Is the **Group Mention** field the slug field?

Reply:
I'm happy with the changes, but want to confirm this as it may be confusing to customers who want to change the slug and don't know where/how.